src/index.html

```diff
@@ -22,6 +22,397 @@
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="Apple iMac"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__text-code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <p class="card__text-review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__text-price">Price:</p>
+          <p class="card__text-value">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="Apple iMac"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__text-code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <p class="card__text-review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__text-price">Price:</p>
+          <p class="card__text-value">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="Apple iMac"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__text-code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <p class="card__text-review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__text-price">Price:</p>
+          <p class="card__text-value">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+        >
+          Buy
+        </a>
+      </div>
```

Comment:
The product cards are repeated multiple times with the same content. Consider using a templating engine or JavaScript to dynamically generate these cards if they are meant to display different products. This will reduce redundancy and make the code easier to maintain.